Formula/erlang.rb

```diff
@@ -19,18 +18,23 @@ class Erlang < Formula
     sha256 cellar: :any, mojave:        "fbb11e617a4031b77dbc4c2d20ff1c8e471a8f9894959e015bdf8ebc0403268e"
   end
 
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
-  depends_on "libtool" => :build
+  head do
+    url "https://github.com/erlang/otp.git"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+  end
+
   depends_on "openssl@1.1"
   depends_on "wxmac" # for GUI apps like observer
 
   uses_from_macos "m4" => :build
 
```

Comment:
We should be able to remove this. (Or find out quickly if we can't.)

Reply:
I can remove and restart run.

How is Linux side for these?

Reply:
Should be fine. Most likely needed for `libtool`, but that's now `head`-only, and there have been recent-ish fixes to `superenv` that make declaring an `m4` dep for `libtool` no longer necessary.

Reply:
Okay. Updated with all changes.

Reply:
Actually, I was wrong. We won't find out so quickly. But it's likely this isn't needed.

Reply:
Triggered a Linux build to find out: https://github.com/carlocab/homebrew-personal/runs/2624511175?check_suite_focus=true

We can put it back in a syntax-only PR if I was wrong.

Reply:
Linux build failed, but for spurious reasons. (`libdrm` bottle missing.)

Reply:
Okay.

I do see that `m4` would probably only impact `autoconf(head)` build and dependency chain should pull it in for Linux.

As ARM node has succeeded, hopefully everything else is good to go.